controllers/clusterversion_controller.go

```diff
@@ -436,3 +450,42 @@ func (c *ClusterVersionReconciler) ensureConsolePlugin() error {
 
 	return nil
 }
+
+func (c *ClusterVersionReconciler) getDeployCSIConfig() (bool, error) {
+
+	// verify existence of storagecluster crd
+	storageClusterCRD := &metav1.PartialObjectMetadata{}
+	storageClusterCRD.SetGroupVersionKind(
+		extv1.SchemeGroupVersion.WithKind("CustomResourceDefinition"),
+	)
+	storageClusterCRD.Name = "storageclusters.ocs.openshift.io"
+	if err := c.get(storageClusterCRD); kerrors.IsNotFound(err) {
+		// storagecluster crd doesn't exist and we need to proceed with deploying CSI
+		return true, nil
```

Comment:
This is wrong. You are ignoring the option that the config map states "false"

Reply:
done.